README.md

```diff
@@ -0,0 +1,24 @@
+# Gnosis Safe Documentation
+
+The Gnosis Safe is a multisignature wallet with support for confirmations using signed messages based on ERC191. It is the successor of the Gnosis Multisig Wallet and combines more functionality with reduced gas costs. The Gnosis Safe allows basic wallet configuration like adding and removing owners and more advanced features like modules, which allow to do transactions with different requirements.
+
```

Comment:
Add something like the possible integrations like the 2FA mobile, extension and browser?

Reply:
I will copy something from some blogpost from @tschubotz 😄